src/components/elementView/powerView/bladePowerView.tsx

```diff
@@ -273,36 +278,15 @@ export class BladePowerView extends React.PureComponent<
     return (
       <div className={Classes.DARK + " propsview"}>
         <h3>Power Connections</h3>
-        {this.state.statusLoaded ? (
-          this.props.asset &&
-          this.props.asset.chassis &&
-          this.props.asset.chassis.model &&
-          this.props.asset.chassis.model.vendor &&
-          this.isBladePowerNetworkControlled() ? (
-            this.renderPowerTable()
-          ) : (
-            this.renderNoPowerCallout()
-          )
-        ) : (
-          <Spinner />
```

Comment:
moved spinner logic to inside the row that's loading instead of replacing the whole table